rest/config_startup.go

```diff
@@ -145,8 +138,8 @@ type ReplicatorConfig struct {
 }
 
 type UnsupportedConfig struct {
-	StatsLogFrequency *base.ConfigDuration `json:"stats_log_frequency,omitempty" help:"How often should stats be written to stats logs"`
-	UseStdlibJSON     bool                 `json:"use_stdlib_json,omitempty"     help:"Bypass the jsoniter package and use Go's stdlib instead"`
+	StatsLogFrequency base.ConfigDuration `json:"stats_log_frequency,omitempty" help:"How often should stats be written to stats logs"`
```

Comment:
I'm not following how we differentiate between stats_log_frequency being absent vs. set to zero with this change.

Reply:
Yeah you're right. I shouldn't have changed this to a non-pointer for that reason.